README.md

```diff
@@ -156,7 +156,27 @@ class CreateUser
 end
 ```
 
-You'll get an `ArgumentError` if you pass any params not defined in the `inputs_for` block, or if you don't supply all required params. This is completely opt-in so if you don't provide an `inputs_for` block, no checking is performed.
+### Specifying types and nullable fields
+
+You can now also optionally specify types and nullability for your inputs. For the `type:` option, any ruby class can be used.
+
+```ruby
+inputs_for(:call) do
+  required :name, type: String, null: false
```

Comment:
Well, then I'm inclined to keep the behavior as is. If you have to put `null: false` on every single `required` field, I think that's a good thing. It's explicit and unambiguous and redundancy is something I will take over implicit and ambiguous every day of the week. It's especially useful for newer developers who aren't familiar with this library.